src/stemmabench/algorithms/stemma.py

```diff
@@ -0,0 +1,209 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union, Any
+from stemmabench.algorithms.manuscript import Manuscript
+from stemmabench.algorithms.manuscript_base import ManuscriptBase
+from stemmabench.algorithms.utils import Utils
+#from stemmabench.algorithms.stemma_algorithm import StemmaAlgo
```

Comment:
remove commented library, you will add them at a later time if necessary

Reply:
Removed comment